src/olympia/reviewers/views.py

```diff
@@ -136,6 +137,101 @@ def beta_signed_log(request):
     return render(request, 'reviewers/beta_signed_log.html', data)
 
 
+@any_reviewer_or_moderator_required
+def dashboard(request):
+    # The dashboard is divided into sections that depend on what the reviewer
+    # has access to, each section having one or more links, each link being
+    # defined by a text and an URL. The template will show every link of every
+    # section we provide in the context.
+    sections = OrderedDict()
+    if acl.action_allowed(request, amo.permissions.ADDONS_REVIEW):
+        sections[ugettext('Legacy Add-ons')] = [(
+            ugettext('New Add-ons ({0})').format(
+                ViewFullReviewQueue.objects.count()),
+            reverse('reviewers.queue_nominated')
+        ), (
+            ugettext('Add-on Updates ({0})').format(
+                ViewPendingQueue.objects.count()),
+            reverse('reviewers.queue_pending')
+        ), (
+            ugettext('Performance'),
```

Comment:
I wonder if Performance should be shown separately as it's applicable to multiple sections?  So you don't get it multiple times if you have more than one permission.

Reply:
I thought about that and I'm open to changing that, but I couldn't figure out an appropriate section title, so it looked kinda weird on its own. We can tweak that later.